license_manager/server/main.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"flag"
+	"net"
+	"strconv"
+
+	lmpb "github.com/enfabrica/enkit/license_manager/proto"
+	"github.com/enfabrica/enkit/license_manager/service"
+
+	"github.com/golang/glog"
```

Comment:
FYI: the repo is currently using the plain log library from golang, OR uber zap library. Given that libraries in the repo are supposed to be usable from any app, most often they rely on the simple lib/logger/Logger interface (but lose on semantic/field based logging, but we could add interfaces).

Here, I'd recommend using either plain log from golang, or our internal wrapper around zap library (lib/logger/klog), I would not bring in a 3rd logging library, unless it has some significant benefit.

Reply:
No more logger since `server.Run` is used, which emits the log message.